aemedge/blocks/accordion/accordion.css

```diff
@@ -31,70 +26,80 @@
   line-height: normal;
   margin: 0;
   color: rgb(23 23 37);
-  border-top: 1px solid transparent;
-  border-right: 1px solid transparent;
-  border-left: 1px solid transparent;
-  border-bottom: none;
-  border-radius: 4px;
-  transform: translate3d(0, 0, 0);
-  transition: color 0.3s ease-in-out 0s;
 }
 
-  /* If you want to add something for when the details are open */
-
-  /* .accordion details[open] summary {
-
-  } */
-
-  /* If you want to add something for focus/hover */
-
-  /* .accordion details summary:focus,
-  .accordion details summary:hover {
-  } */
+.accordion .details.open .summary {
+  font-weight: bold;
+  transition: color 0.3s ease-in-out 0s;
+}
 
-.accordion details summary::-webkit-details-marker {
+.accordion .details .summary::-webkit-details-marker {
   display: none;
 }
 
-.accordion details summary::before {
-  content: "";
+.accordion .details .summary::before {
+  content: '';
   position: relative;
   color: rgb(23 23 37);
-  background: url("../../icons/accordion-arrow-down.svg") no-repeat;
+  background: url('../../icons/accordion-arrow-down.svg') no-repeat;
   width: 16px;
   height: 16px;
-  margin-right: 10px;
+  margin: 4px 1.5rem 0 1rem;
+  transition: transform 0.5s ease 0s;
 }
 
-.accordion details[open] summary::before {
-  transform: translateY(-50%) rotate(-180deg);
-  margin-top: 8px;
+.accordion .details.open .summary::before {
+  transform: rotate3d(1, 0, 0, -180deg);
+  transition: transform 0.5s ease 0s;
 }
 
-.accordion details summary p {
+.accordion .details .summary p {
   margin: unset;
 }
 
-.accordion details .accordion-item-body {
+.accordion .details .accordion-item-body {
   text-align: left;
-  padding: 0 16px;
-}
-
-.accordion details[open] .accordion-item-body {
   padding-right: 4.25rem;
+  height: auto;
+  max-height: 0;
+  overflow: hidden;
+  padding-left: 55px;
+  transition: max-height 500ms ease-in-out 0s;
 }
 
-.accordion details[open] .accordion-item-body ul {
+.accordion.details.open .accordion-item-body ul {
   margin-left: 40px;
 }
 
-.accordion details[open] .accordion-item-body ul li {
+.accordion.details.open .accordion-item-body ul li {
   padding: 0.5rem 0;
 }
 
 /* width greater than 768px */
 @media (width >= 768px) {
-  .accordion details[open] .accordion-item-body ul li {
+  .accordion .details .accordion-item-body ul li {
     padding: 0.5rem 4rem 0.5rem 0;
   }
-}
\ No newline at end of file
+
+  .accordion .details.open .accordion-item-body ul li {
+    padding: 0.5rem 4rem 0.5rem 0;
+  }
+
+  .accordion .details .accordion-item-body {
+    padding-left: 65px;
+  }
+
+  .accordion .details.open .accordion-item-body {
+    padding-left: 65px;
+  }
+
+  .accordion .details .summary::before {
+    margin-left: 1.5rem;
+  }
+
+  .accordion-wrapper {
+    margin: 0 auto;
```

Comment:
Can you change this margin to
`margin: 0 auto 4rem auto;`
it will give needed spacing at the bottom.